I.pass-tests/05-factorial/factorial.js

```diff
@@ -8,6 +8,14 @@
 // calculate and return the factorial of int
 // note: factorial of 0 is 1
 
-function factorial(int) {}
+function factorial(int) {
+    // let result = 1, i = int;
+    if (int === 0 || int === 1)
```

Comment:
while int is not reserved anymore in JS 5, it is reserved word in previous versions afaik
it's also reserved in many other languages. so while parameter is coming as int, it can be useful to save it in another variable

Reply:
Also this will save you the trouble of the first conditions, as you can initialize it with 1